docs/develop/external-calls.md

```diff
@@ -1,28 +1,29 @@
 ---
 title: 'External API call support in Office Scripts'
 description: 'Support and guidance for making external API calls in an Office Script.'
-ms.date: 09/24/2020
+ms.date: 12/28/2020
 localization_priority: Normal
 ---
 
 # External API call support in Office Scripts
 
-The Office Scripts platform doesn't support calls to [external APIs](https://developer.mozilla.org/docs/Web/API). However, these calls can be run under the right circumstances. External calls can be only be made through the Excel client, not through Power Automate [under normal circumstances](#external-calls-from-power-automate).
+Script authors shouldn't expect consistent behavior when using [external APIs](https://developer.mozilla.org/docs/Web/API) during the platform's preview phase. This is due how the JavaScript runtime manages interacting with the workbook. The script may end before the API call completes (or its `Promise` is fully resolved). As such, do not rely on external APIs for critical script scenarios.
 
-Script authors shouldn't expect consistent behavior when using external APIs during the platform's preview phase. This is due how the JavaScript runtime manages interacting with the workbook. The script may end before the API call completes (or its `Promise` is fully resolved). As such, do not rely on external APIs for critical script scenarios.
+Calls to external APIs can be only be made through the Excel client, not through Power Automate [under normal circumstances](#external-calls-from-power-automate).
 
 > [!CAUTION]
 > External calls may result in sensitive data being exposed to undesirable endpoints. Your admin can establish firewall protection against such calls.
 
-## Definition files for external APIs
+## Working with `fetch`
 
-The definition files for external APIs aren't included with Office Scripts. The use of such APIs generates compile-time errors for missing definitions. The APIs still run (though only when run through the Excel client), as shown in the following script:
+The [fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) retrieves information from external services. It is an `async` API, so you will need to adjust the `main` signature of your script. Make the `main` function `async` and have it return a `Promise<void>`.
```

Comment:
re; '... return a Promise<void>'

This is correct in the basic case, though the return type may evolve in PA scenarios. I'll leave it to you if that's intentional here.